researchstudio-sat/webofneeds
# Fix #2458 #2469

Fixes #2458 
Fixes #2469 

Adds a type to the connection Stored in the state, but does not use the correctType yet (this will be fixed in a sep. issue see #2482)

How To Test:
Create a Persona, and without reload add a new Need with that Persona attached -> one should see the Persona in the post-content-general part of the need-detail view now.
